### ISSUE TITLE
Question about communication sizes

### ISSUE THREAD

**Issue by dnat112** [mc2-project/delphi#7]
I am trying to use your library to measure the the communication size of delphi. I am able to use your provided script bandwidth_client.sh with wireshark to get the overall communication size. However, I am not sure how to separate this size into the online vs offline/pre-computation costs, as reported in your papers. Could you provide instructions on how to go about doing this?

Thank you!

**Comment by ryanleh** [comment]
There's actually a much easier way of measuring communication (by directly counting the bytes sent in Rust via something like [this](https://github.com/ryanleh/io-utils/blob/main/src/counting.rs)) than what the original script did. Since there is a demand for it, I will try and integrate these into delphi soon (hopefully latest is next week).

**Comment by ryanleh** [comment]
Hi, I've been a bit more swamped then expected so haven't had time to get to this. If you would like, I just remembered that I have already implemented something along the lines of what you're looking for in the `muse-experiments` branch. In this branch the latency experiments count and display the bandwidth used (example [here](https://github.com/mc2-project/delphi/blob/891c16a5c9d2fad8b797ab56cb74a9bdd64659e3/rust/experiments/src/latency/client.rs#L63)).
